ui/src/components/group/GroupReviewTable.js

```diff
@@ -81,14 +83,24 @@ class GroupReviewTable extends React.Component {
     constructor(props) {
         super(props);
         this.submitReview = this.submitReview.bind(this);
+        this.updateReviewGroup = this.updateReviewGroup.bind(this);
+        this.onClickDeleteCancel = this.onClickDeleteCancel.bind(this);
         this.onUpdate = this.onUpdate.bind(this);
         let members = props.members && props.members.map((m) => m.memberName);
         this.state = {
             submittedReview: false,
+            showDeleteConfirmation: false,
             extendedMembers: new Set(members),
             deletedMembers: new Set(),
         };
     }
+
+    componentDidUpdate = (prevProps) => {
+        if (prevProps.members.length > this.props.members.length) {
```

Comment:
I don't think relying on the length is a good indication whether the data has changed or not

Reply:
In my opinion, if there are more members in the prevProps.members, is a good indication that at least one member was deleted, so now, the deleted members set contains a member that has already been deleted. Anyway, as you mentioned in the comment below, we can get rid of this code since we are routing to the members page through onUpdateSuccess.